Stripe.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -24,8 +24,7 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/erikdoe/ocmock",
       "state" : {
-        "branch" : "master",
-        "revision" : "05cbc9d934e84ad32530fa53fd7d29c7966d5828"
+        "revision" : "2c0bfd373289f4a7716db5d6db471640f91a6507"
```

Comment:
This hash corresponds to the 3.9.4 release tag:
https://github.com/erikdoe/ocmock/releases/tag/v3.9.4

Swift Package manager doesn't allow OCMock unless you add it using a commit hash:
https://github.com/erikdoe/ocmock/issues/500#issuecomment-955731638

Reply:
This is what caused all of our Xcode 15 tests to fail on Bitrise while only the Xcode 16 tests pass